themes/aframe/source/css/home.styl

```diff
@@ -50,8 +51,9 @@ ul
 
 .home__nav
   display flex
-  li
-    margin-right 10px
+  margin 30px $padding-med $padding-med 0
```

Comment:
I use it in a bunch of places.  I'll go with _spacing_ for now and we can figure out better naming a bit later.  Thanks!